site/src/pages/tutorials/tutorial-blog-rest-step3.mdx

```diff
@@ -0,0 +1,169 @@
+
+1. Declare a service method, `createBlogPost()`, in the class `BlogService`.
+2. [Map this service method](/docs/server-annotated-service#mapping-http-service-methods) with the HTTP POST by adding the <type://@Post> annotations.
+3. Assign the endpoint as `/blogs`.
+
+```java filename=BlogService.java highlight=1
+@Post("/blogs")
+public HttpResponse createBlogPost(BlogPost blogPost) throws JsonProcessingException {}
+```
+
+## 2. Handle parameters
+
+Let's take post information through a request body. Armeria's [request converter](/docs/server-annotated-service#converting-an-http-request-to-a-java-object) converts request parameters in HTTP messages into Java objects for you. In the request converter we define what keys of a JSON object to map with what properties of a Java object.
+
+To handle parameters received for creating a blog post, use <type://RequestConvert>, in these steps. The order may be changed.
+
+- [Write a request converter](#write-a-request-converter)
+- [Assign a request converter](#assign-a-request-converter)
+
+### Write a request converter
+
+Armeria's [request converter](../docs/server-annotated-service#converting-an-http-request-to-a-java-object) converts a request body from a client into a Java object for you.
+
+We can use Armeria's default request converter as is, but here let's give a go at customizing a request converter for our blog post requests. We want to convert blog post details into a Java object.
+
+1. Create a BlogPostRequestConverter.java file.
+2. Declare a class in the file, implementing the <type://RequestConverterFunction> interface. For simplicity sake, generate impromptu IDs in this tutorial.
+  ```java filename=BlogRequestConverter.java
+  final class BlogPostRequestConverter implements RequestConverterFunction {
+    private static final ObjectMapper mapper = new ObjectMapper();
+    private final AtomicInteger idGenerator = new AtomicInteger(); // Blog post ID
+  }
+  ```
+3. Add a method retrieving a value of a given key in a JSON object:
+  ```java filename=BlogRequestConverter.java highlight="4"
+    static String stringValue(JsonNode jsonNode, String field) {
+      final JsonNode value = jsonNode.get(field);
+      if (value == null) {
+          throw new IllegalArgumentException(field + " is missing!");
+      }
+      return value.textValue();
+    }
+  ```
+4. Customize the default `convertRequest()` method as follows.
+    - Map the `title` and `content` properties of the body parameter to members in a `BlogPost` object, using the `stringValue()` method we wrote in the previous step.
+    - Throw the <type://FallthroughException> by calling <type://RequestConverterFunction#fallthrough()>, to pass on the request parameters that aren't converted by our converter to the default request converter.
+    - Feel free to add more mapping blocks (lines 9 to 11) in this request converter for other requests for the blog service.
```

Comment:
Perhaps we don't need this sentence because it can confuse a user?

Reply:
Done https://github.com/line/armeria/pull/3420/commits/3949b2b3d73d89f5658908a153d1d94694ff43bd